dist/list/list.css

```diff
@@ -0,0 +1,84 @@
+.list {
+    max-width: 480px;
+}
+
+.list ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list ul li > * {
+    align-items: center;
+    background-color: var(
+        --list-background-color,
+        var(--color-background-primary)
+    );
+    box-sizing: border-box;
+    color: var(--color-foreground-on-primary);
+    display: inline-flex;
+    font-size: var(--font-size-16);
+    margin: 1px;
+    min-height: var(--spacing-600);
+    padding: var(--spacing-150) var(--spacing-200);
+    width: 100%;
+}
+
+.list__leading {
+    margin-right: var(--spacing-200);
+}
+
+.list__trailing {
+    margin-left: var(--spacing-200);
+}
+
+.list__body {
+    flex: 1;
+}
+
+.list li > a,
```

Comment:
Probably negligible, but if you use `>` here instead of a general child selector it should improve performance and also prevent this from effecting nested lists

Reply:
Maybe
But I dont think there needs to be other lists inside. So that's why i didnt have it as a direct descendant